extensions/amp-analytics/0.1/requests.js

```diff
@@ -98,15 +99,15 @@ export class RequestHandler {
           this.variableService_.expandTemplate(this.baseUrl, expansionOption);
       this.baseUrlPromise_ = this.baseUrlTemplatePromise_.then(baseUrl => {
         return this.urlReplacementService_.expandAsync(
-            baseUrl, undefined, this.whiteList_);
+            baseUrl, dynamicBindings, this.whiteList_);
```

Comment:
Just confirming: dynamicBindings can be a string or a Promise...expandAsync can handle this?

Reply:
Yes